examples/mimas2/run_yosys.ys

```diff
@@ -0,0 +1,4 @@
+read_verilog example.v
+synth_xilinx -top example -family xc6s
+iopadmap -bits -outpad OBUF I:O -inpad IBUF O:I
```

Comment:
Perhaps I was too hasty to merge it in. Would it be useful to have this `iopadmap` command be part of `synth_xilinx -family xc6s`?

Reply:
To be discussed in #1235